# Try & catch

La instrucción try se utiliza para que esté al pendiente de que suceda algun error al momento de ejecturse el código. Si sucede alguna falla se suspende el *try* y se ejecuta la instrucción *catch*, en otras palabras, *try* es remplazada por *catch*. Estas dos intrucciones van de la mano. Si no se hace uso de estas instucciones, simplemente el programa marcaría error y deja de correr. 

**Validar**: Validar es asegurarse de que no se rompa el programa, de que cada error que pueda estar presente en el programa tenga por detrás a *try* & *catch*.

```
x = 15
y = 0
try:
	z = x/y
	"El resultado de dividir " + x + " entre" + y + "es igual a " + z + "."
Catch
	"Error: no existe la división entre 0."
```

## Referencias

- [Belisa, A.](https://platzi.com/profesores/anabelisam_/) (s.f.). _Try Catch_. [Nuevo Curso de Pensamiento Lógico: Manejo de Datos, Estructuras y Funciones](https://platzi.com/cursos/pensamiento-logico-estructuras/). Recuperado el 16 de agosto de 2022 de https://platzi.com/clases/3222-pensamiento-logico-estructuras/50897-try-catch/

- [Belisa, A.](https://platzi.com/profesores/anabelisam_/) (s.f.). _Reto 7: algoritmos con manejo de errores_. [Nuevo Curso de Pensamiento Lógico: Manejo de Datos, Estructuras y Funciones](https://platzi.com/cursos/pensamiento-logico-estructuras/). [Platzi](https://www.platzi.com/home). Recuperado el 16 de agosto de 2022 de https://platzi.com/clases/3222-pensamiento-logico-estructuras/50898-reto-7-algoritmos-con-manejo-de-errores/